src/ensembl/stories/shared-components/layout/Layout.stories.tsx

```diff
@@ -85,6 +87,7 @@ const Wrapper = (props: {
         sidebarNavigation={<SidebarNavigation />}
         sidebarToolstripContent={<SidebarToolstripContent />}
         drawerContent={props.drawerContent || null}
+        sidebarBehaviour={props.sidebarBehaviour || SidebarBehaviourType.PUSH}
```

Comment:
Minor consideration (feel free to ignore it if you don't like it). Probably a fairer implementation (reflecting the optionality of this prop) would be something like:

```
<StandardAppLayout
  //stuff
  {... (props.sidebarBehaviour ? {sidebarBehaviour: props.sidebarBehaviour} : null ) } 
/>
```
This would produce an object `{sidebarBehaviour: props.sidebarBehaviour}` if there is `props.sidebarBehaviour`, or an empty object otherwise. Which will prove that StandardAppLayout can work without any sidebarBehaviour property at all.

Reply:
I did this to the other two default props as well: `isDrawerOpen` and `onDrawerClose`.